packages/angular_devkit/build_angular/src/builders/application/index.ts

```diff
@@ -83,12 +85,30 @@ export async function* buildApplicationInternal(
 
   yield* runEsBuildBuildAction(
     async (rebuildState) => {
+      const { prerenderOptions, outputOptions } = normalizedOptions;
+
       const startTime = process.hrtime.bigint();
       const result = await executeBuild(normalizedOptions, context, rebuildState);
 
+      if (prerenderOptions) {
+        const prerenderedRoutesLength = result.prerenderedRoutes.length;
+        let prerenderMsg = `Prerendered ${prerenderedRoutesLength} static route`;
+        prerenderMsg += prerenderedRoutesLength !== 1 ? 's.' : '.';
+
+        result.addInfo(ansiColors.magenta(prerenderMsg));
+      }
+
       const buildTime = Number(process.hrtime.bigint() - startTime) / 10 ** 9;
-      const status = result.errors.length > 0 ? 'failed' : 'complete';
-      logger.info(`Application bundle generation ${status}. [${buildTime.toFixed(3)} seconds]`);
+      const hasError = result.errors.length > 0;
+      if (writeToFileSystem && !hasError) {
+        result.addInfo(`Output location: ${outputOptions.base}\n`);
```

Comment:
@clydin, what do you prefer the path to be here absolute or relative to the CWD?

Reply:
I am leaning towards relative to CWD to avoid system specific information in logs as well as most likely shorter paths.
However, i think there should be an exception for the case of the base already being absolute. In that case, I would leave it as specified.
This is also in a prerelease so we can alter it before the final if needed.